packages/dds/merge-tree/tsconfig.json

```diff
@@ -6,7 +6,8 @@
     "compilerOptions": {
         "rootDir": "./src",
         "outDir": "./dist",
-        "composite": true
+        "composite": true,
+        "preserveConstEnums": true
```

Comment:
why do we want this? it sounds like it will increase the memory usage of the merge tree code: https://www.typescriptlang.org/tsconfig#preserveConstEnums

Reply:
as all references are still inlined, but then we also emit the enum declaration, but no one will use it

Reply:
you can actually see this increase in the bundle size analysis above

Reply:
i should have read the bug :)